### DISCUSSION TITLE
Approval Request for Future State Proposal Implementation

### DISCUSSION THREAD

**Discussion by vvalderrv** [NativeScript/NativeScript#10502] [General]
The Linux Foundation is seeking approval to begin implementing the proposal outlined in the document linked below:

[NativeScript Future State](https://docs.google.com/presentation/d/1pbCEi2y3m8gwATb19O7Zu8gceCt2MoyBfyH_pafsh2w/edit?usp=sharing)

Please review the proposal and sign off on the plan by replying with your approval.

If you have any further questions, please review the [Transition FAQ](https://docs.google.com/document/d/1gESWS5brgxXm9EQrxctmievRedyYp9Q0dtHxLEcL1nk/edit?usp=sharing) and feel free to reach out directly.

Thank you for your attention to this matter.

Vanessa Valderrama
LF Release Engineering
vvalderrama@linuxfoundation.org

**Comment by NathanWalker** [comment]
@vvalderrv Approved.

**Reply by NathanWalker** [reply]
We can go ahead and move forward @vvalderrv, this was discussed in the TSC meeting and channels, all clear ✅
